browser/components/CodeEditor.js

```diff
@@ -141,6 +141,10 @@ export default class CodeEditor extends React.Component {
       this.editor.setOption('indentWithTabs', this.props.indentType !== 'space')
     }
 
+    if (prevProps.lineNumber !== this.props.lineNumber) {
+      this.editor.setOption('lineNumbers', this.props.lineNumber)
+    }
```

Comment:
I think you can set even if `prevProps.lineNumber !== this.props.lineNumber` is false.
Is it for performance?

Reply:
Yes, it is and as you can see, the checks are performed for all the options in the [componentDidUpdate](https://github.com/xxdavid/Boostnote/blob/7a4258bb208d43be80e575fe61c0a0a187811e3f/browser/components/CodeEditor.js#L117) method.